### PR TITLE
More cutting words in more languages

### PR DESCRIPTION
**Pull request by teolemon**
More cutting words

### CODE REVIEW

File: lib/ProductOpener/Ingredients.pm [hunk 1]
```diff
@@ -918,10 +916,13 @@ de => [
 'Vorbereitung Tipps',
 'Mindestens altbar bis',
 'Mindestens haltbar bis',
+'N(â|a|ä)hrwertinformation je 100 ml',
 'Durchschnittliche N(â|a|ä)hrwerte',
 'davon ges(â|a|ä)ttigte',
 'Nâhrwerte',
 'k(u|ü)hl und trocken lagern',
+'Ungeöffnet bei max.',
```
Comment (teolemon) [inline lib/ProductOpener/Ingredients.pm]:
```suggestion
'Unge(ö|o)ffnet bei max.', 
```

File: lib/ProductOpener/Ingredients.pm [hunk 2]
```diff
@@ -705,6 +705,8 @@ fr => [
 'ingr(e|é)dients(\s*)(-|:|\r|\n)+',	# need a colon or a line feed
 'Quels Ingr(e|é)dients ?', # In Casino packagings
 'ingr(e|é)dient(\s*)(-|:|\r|\n)+',
+'ingr(e|é)dienits(\s*)(-|:|\r|\n)+',
```
Comment (teolemon) [inline lib/ProductOpener/Ingredients.pm]:
```suggestion
#'ingr(e|é)dienits(\s*)(-|:|\r|\n)+',
```

File: lib/ProductOpener/Ingredients.pm [hunk 3]
```diff
@@ -10,7 +10,7 @@
 # published by the Free Software Foundation, either version 3 of the
 # License, or (at your option) any later version.
 #
-# This program is distributed in the hope that it will be useful,
+# This program is distributed in the hope that it will be useinfful,
```
Comment (hangy) [inline lib/ProductOpener/Ingredients.pm]: 😕

**Comment by teolemon** [comment]
corrigé @stephanegigandet

File: lib/ProductOpener/Ingredients.pm (hunk 3 above)
Comment (teolemon) [reply lib/ProductOpener/Ingredients.pm]: it's late, I'm correcting it :-)

Comment (teolemon) [inline lib/ProductOpener/Ingredients.pm]:
```suggestion
# This program is distributed in the hope that it will be useful,
```

Comment (teolemon) [reply lib/ProductOpener/Ingredients.pm]: done

**Comment by teolemon** [comment]
<pre>syntax error at /home/travis/build/openfoodfacts/openfoodfacts-server/lib/ProductOpener/Ingredients.pm line 1042, near &quot;'Durchschnittliche N(â|a|ä)hrwert(angaben|angabe)'&quot;

**Comment by teolemon** [comment]
<pre>syntax error at /home/travis/build/openfoodfacts/openfoodfacts-server/lib/ProductOpener/Ingredients.pm line 1044, near &quot;'N(â|a|ä)hrwerte je'&quot;

**Comment by teolemon** [comment]
@stephanegigandet @hangy the build is green again

**Pull request merged**